docs/tutorials/1 - Beginner - Single Module Example.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # 1 - Beginner - Single Module Example
+# 
+# This tutorial shows how to assign an array of irradiances as inputs to a module. It is assigning 12 values of irradiances Gpoat, 1 value to each row of six cells in a 12 x 6 module (72 cell module). 
+
+# In[1]:
+
+
+import pvmismatch  # this imports everything we need
+import numpy as np
+import seaborn as sns
+import pandas as pd
+
+
+# #### Inputs
+
+# In[2]:
+
+
+## Inputs:
+numcells = 72
+Gpoat = [0.9, 0.9, 0.8, 0.7, 0.7, 0.8, 0.9, 0.9, 0.9, 0.9, 0.9, 0.9] # kW/m2 units
+portraitorlandscape = 'portrait'
+
+
+# #### Select module type
+# 
+# The stdpl matrix shows the placement of the cells in the module. The modules we are using are the standard PVMismatch modules, look at the references for the bypass diode groups, but because of this it does matter if the module is in ladscape or portrait.
+
+# In[3]:
+
+
+# cell placement for 'portrait'.
+if numcells == 72:
+    stdpl=np.array([[0,	23,	24,	47,	48,	71],
+    [1,	22,	25,	46,	49,	70],
+    [2,	21,	26,	45,	50,	69],
+    [3,	20,	27,	44,	51,	68],
+    [4,	19,	28,	43,	52,	67],
+    [5,	18,	29,	42,	53,	66],
+    [6,	17,	30,	41,	54,	65],
+    [7,	16,	31,	40,	55,	64],
+    [8,	15,	32,	39,	56,	63],
+    [9,	14,	33,	38,	57,	62],
+    [10,	13,	34,	37,	58,	61],
+    [11,	12,	35,	36,	59,	60]])
+
+elif numcells == 96:
+    stdpl=np.array([[0,	23,	24,	47,	48,	71,	72,	95],
+        [1,	22,	25,	46,	49,	70,	73,	94],
+        [2,	21,	26,	45,	50,	69,	74,	93],
+        [3,	20,	27,	44,	51,	68,	75,	92],
+        [4,	19,	28,	43,	52,	67,	76,	91],
+        [5,	18,	29,	42,	53,	66,	77,	90],
+        [6,	17,	30,	41,	54,	65,	78,	89],
+        [7,	16,	31,	40,	55,	64,	79,	88],
+        [8,	15,	32,	39,	56,	63,	80,	87],
+        [9,	14,	33,	38,	57,	62,	81,	86],
+        [10,	13,	34,	37,	58,	61,	82,	85],
+        [11,	12,	35,	36,	59,	60,	83,	84]])
+
+if portraitorlandscape == 'landscape':
+    stdpl = stdpl.transpose()
+
+cellsx = len(stdpl[1]); cellsy = len(stdpl)
+    
+
+
+
+# #### Let's create the type of module we want
+
+# In[4]:
+
+
+if cellsx*cellsy == 72:
+    cell_pos = pvmismatch.pvmismatch_lib.pvmodule.STD72
+elif cellsx*cellsy == 96:
+    cell_pos = pvmismatch.pvmismatch_lib.pvmodule.STD96
+
+pvmod=pvmismatch.pvmismatch_lib.pvmodule.PVmodule(cell_pos=cell_pos)
+
+
+# #### Let's make the system be just 1 module
+
+# In[5]:
+
+
+pvsys = pvmismatch.pvsystem.PVsystem(numberStrs=1, numberMods=1, pvmods=pvmod)  
+
+
+# #### Create the pattern of irradiance based on the Gpoat input. 
+# 
+# We are assigning the gradient across the module for this case. 
+
+# In[6]:
+
+
+G=np.array([Gpoat]).transpose()
+H = np.ones([1,cellsx]) 
+array_det = np.dot(G,H) 
+sns.heatmap(array_det, square = True)
```

Comment:
![image](https://user-images.githubusercontent.com/19290930/84713920-0754c900-af21-11ea-89d6-ae2b849f5a5f.png)

Reply:
@shirubana 
Seaborn is currently not in the "requirements.txt". We can choose from these options. 
1- Either add it as a dependency
2- make the change to Matplotlib's native matshow to show the same sort of heat map.

It may be preferred to keep the package with as few dependencies as possible but I'd leave it to your discretion.